mifosng-provider/src/main/resources/sql/migrations/core_db/V249__bug_fix_teller_cash_management.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE `m_permission` SET `action_name`='UPDATE' WHERE  `id`=627;
```

Comment:
this query will not work on all databases (id may refer to different permissions)